kitsune/messages/jinja2/messages/outbox.html

```diff
@@ -17,42 +17,27 @@ <h1 class="sumo-page-heading">{{ title }}</h1>
           <div class="sumo-button-wrap">
             <button type="submit" name="delete" class="sumo-button">{{ _('Delete Selected') }}</button>
           </div>
-          <ol class="message-list">
-              <li class="message-list--item">
-                <div class="field checkbox no-label">
+        <div class="outbox-table">
+            <div class="email-row header">
+                <div class="email-cell checkbox">
+                    <input type="checkbox" aria-label="Select all">
                 </div>
-                <div class="avatar-details">
-                  <span class="to user">
-                    <b>{{ _('Sent') }}</b>
-                  </span>
-                  <span class="to user">
-                    <b>{{ _('To') }}</b>
-                  </span>                  
-                  {% if in_staff_group(request.user) %}
-                    <span class="to group">
-                        <b>{{ _('To Groups') }}</b>
-                    </span>
-                  {% endif %}
-                <span class="message-view">
-                    <b>{{ _('Message') }}</b>
-                </span>
-                <span class="delete-button">
-                </span>
-                </div>
-              </li>
+                <div class="email-cell sent">{{ _('Sent On') }}</div>
+                <div class="email-cell to">{{ _('To Users') }}</div>
+                {% if in_staff_group(request.user) %}
+                    <div class="email-cell to-groups">{{ _('To Groups') }}</div>
+                {% endif %}
+                <div class="email-cell excerpt">{{ _('Excerpt') }}</div>
+            </div>
```

Comment:
For consistency with the inbox, can we add an individual delete column here? If so, could we then remove the `Delete Selected` button above?